test/language/class/class-strict-mode.js

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) Copyright 2014 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 14.5
+description: >
+    class strict mode
```

Comment:
The top level with was used to ensure that the outer context of the class was sloppy.